ci-operator/step-registry/rosa/e2e/ocm-fvt/rosa-e2e-ocm-fvt-commands.sh

```
@@ -0,0 +1,66 @@
+
+env -i bash --norc --noprofile -c '
+  source /usr/local/cs-qe-credentials/ocm-tokens
+  source /usr/local/cs-qe-credentials/jira-cred
+  env | grep -v "^_="
+' >> "${podman_env_file}"
+
+podman_args=(
+  --authfile /usr/local/cs-qe-credentials/.dockerconfigjson
+  --env-file "${podman_env_file}"
+  -v /usr/local/cs-qe-credentials:/credentials:ro,z
+)
```

Shellcheck (0.11.0)

[warning] Use spaces, not commas, to separate array elements.

(SC2054)